.drone.yml

```diff
@@ -0,0 +1,49 @@
+workspace:
+  base: /test
+  path: openeuropa-site
+
+services:
+  web:
+    image: fpfis/httpd-php-dev:7.1
+    environment:
+      - DOCUMENT_ROOT=/test/openeuropa-site
+      - COMPOSER_CACHE_DIR=/cache/composer
+  mysql:
+    image: percona/percona-server:5.6
+    environment:
+      - MYSQL_ALLOW_EMPTY_PASSWORD=yes
+  selenium:
+    image: selenium/standalone-chrome:3.11
+    environment:
+      - DISPLAY=:99
+      - SE_OPTS=-debug
+
+pipeline:
+  # Download all dependencies.
```

Comment:
The test with lower dependencies is missing.

Reply:
Now, the test with higher dependencies is missing. ;)

Reply:
See line 30